docs/install.md

```diff
@@ -62,3 +58,75 @@ See the docs to help determine proper sizes:
 
 For example: you have a workload that requires 15 pods and you want to run 2 nodes.
 15+45 is a 60 pod minimum requirement, so running 2 t3.large nodes (2*35=70) would be just enough.
+
+## Features
+
+Rio contains several independent components that are responsible for different features. By default It will install all components which give you full functionality including service mesh, source code to deployment, autoscaling and advanced API gateway.
+If you have limited capacities in your cluster and don't want enable some of features, you can disable features to make footprint of Rio smaller.
+
+Here is the list of features:
+
+| Feature     | Description                                       |
+|-------------|---------------------------------------------------|
+| autoscaling | Auto-scaling services based on in-flight requests |
+| build       | Rio Build, from source code to deployment         |
+| gloo        | API gateway backed by gloo                        |
+| linkerd     | Linkerd service mesh                              |
+| letsencrypt | Let's Encrypt                                     |
+| rdns        | Acquire DNS from public Rancher DNS service       |
+| dashboard   | Rio UI                                            |
+
+The core functionality of Rio includes Rio CLI UX and Riofile UX, which is built-in and doesn't rely on any of these features. Beside core functionality, here is a list of advanced features.
+
+**Autoscaling**
+
+This is the feature that manages autoscaling feature in Rio. It deploys a pod inside cluster, watch autoscaling config for each service and scrape metrics from pods. According to real time metrics and autoscaling config, it will make decisions to scale pods up and down.
```

Comment:
```suggestion
Manages autoscaling services in Rio. It deploys a pod inside cluster, watches the autoscaling config for each service, and scrapes metrics from pods. It will then make decisions to scale pods up and down based on those real-time metrics and autoscaling configs.
```